modules/monitoring-granting-users-permission-to-configure-monitoring-for-user-defined-projects.adoc

```diff
@@ -6,7 +6,7 @@
 [id="granting-users-permission-to-configure-monitoring-for-user-defined-projects_{context}"]
 = Granting users permission to configure monitoring for user-defined projects
 
-You can grant users permission to configure monitoring for user-defined projects.
+As a cluster administrator, you can assign the `user-workload-monitoring-config-edit` role to a user to grant permission to configure and manage monitoring for user-defined projects without giving them permission to configure and manage core {product-title} monitoring components.
```

Comment:
```suggestion
As a cluster administrator, you can assign the `user-workload-monitoring-config-edit` role to a user to grant permission to configure and manage monitoring for user-defined projects. This does not give them permission to configure and manage core {product-title} monitoring components.
```

Reply:
(to break up the sentence since it's long)

Reply:
Thank you for the tip. I will definitely break the sentence into two. I will just divide it slightly differently as I want to keep the "without giving" part. The reason is I believe that the suggestion slightly changes the meaning. Because the suggestion slightly implies the admin would want to give all the permissions, but that it not the case. So I will break up the sentence as follows:
```suggestion
As a cluster administrator, you can assign the `user-workload-monitoring-config-edit` role to a user. This grants permission to configure and manage monitoring for user-defined projects without giving them permission to configure and manage core {product-title} monitoring components.
```